src/Entity/Transaction.php

```diff
@@ -23,6 +23,10 @@ class Transaction
     public const PAYBOX_CARD_UNAUTHORIZED = '000021';
     public const PAYBOX_PAYMENT_PAGE_TIMEOUT = '000030';
 
+    public const TYPE_PAYBOX = 'paybox';
+    public const TYPE_CHEQUE = 'cheque';
```

Comment:
why switch to FR for this const? can you stay in ENG pls?

Reply:
in english it's cheque or check when I try google translate. :s

Reply:
lol you are right, cheque is in eng too :D thanks wiki